docs/getting_started.md

```diff
@@ -63,6 +63,14 @@ necessary dependencies
    * `pip install -r requirements-dev.txt`
    * `pre-commit install --hook-type pre-commit --hook-type commit-msg`
 
+
+## OCS-CI Container Image
+  We also have ocs-ci container image available at quay.io/ocsci/ocs-ci, the image
+  is build using the latest branch. It is still experimental as the ci requires
```

Comment:
This says 'latest branch' but it looks like we're pushing this image when the stable branch is updated. Should that be mentioned here instead?

Reply:
that is correct, thanks I will fix it.